test/tests/functional/pbs_test_run_count.py

```diff
@@ -65,6 +65,15 @@ def disable_reject_begin_hook(self):
         self.mom.log_match("h1.HK;copy hook-related file request received",
                            existence=True, starttime=start_time)
 
+    def enable_reject_begin_hook(self):
+        start_time = int(time.time())
```

Comment:
Do you need this start_time variable or can you use the time directly in log_match function if it not reused?

Reply:
Nope, can cause race condition.

Reply:
got it. Thanks!